core/domain/topic_services.py

```diff
@@ -282,8 +282,8 @@ def _save_topic(committer_id, topic, commit_message, change_list):
     topic_model.additional_story_ids = topic.additional_story_ids
     topic_model.skill_ids = topic.skill_ids
     topic_model.language_code = topic.language_code
-    change_list = [change.to_dict() for change in change_list]
-    topic_model.commit(committer_id, commit_message, change_list)
+    change_list_dict = [change.to_dict() for change in change_list]
```

Comment:
This is not a dict. change_dicts perhaps.

Reply:
Done.